roles/addons/tasks/skydns.yml

```diff
@@ -7,9 +7,9 @@
   register: dns_rc_def
   when: dns_setup
   tags:
-    - skydns
+    - addons
```

Comment:
can we keep the skydns tag in addition to addons, that way if someone wants to leverage the tag to specify a specific addon to include?